### PR TITLE
Ability to generate foreman-debug in specific local folder

### PR DESCRIPTION
**Pull request by sthirugn**
- By default the foreman-debug is created in my automation-tools root folder - this PR will give ability to user to create that file in a specific folder.

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1474,7 +1477,7 @@ def foreman_debug(tarball_name=None):
     with cd('/tmp'):
         run('foreman-debug -q -d {0}'.format(tarball_name), quiet=True)
         get(
-            local_path='%(basename)s',
+            local_path=user_path if user_path else '%(basename)s',
```
Comment (omaciel) [inline automation_tools/__init__.py]: I believe you also want to check that `user_path is not None`...

Comment (sthirugn) [inline automation_tools/__init__.py]: Nope. It is not needed. The following tests worked fine for me:
```
fab -H $host foreman_debug:user_path=''
fab -H $host foreman_debug:user_path=""
fab -H $host foreman_debug:user_path=None
```

Comment (omaciel) [inline automation_tools/__init__.py]: ACK

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -1460,12 +1460,15 @@ def update_packages(*args, **kwargs):
 # =============================================================================
 # Utility tasks
 # =============================================================================
-def foreman_debug(tarball_name=None):
+def foreman_debug(tarball_name=None, user_path=None):
```
Comment (elyezer) [inline automation_tools/__init__.py]: s/user_path/local_path?

**Comment by elyezer** [comment]
ACK, pending comment.

**Comment by elyezer** [comment]
APT

**Comment by sthirugn** [comment]
Got two ACKs (one hidden in the diff)

**Comment by elyezer** [comment]
@sthirugn thank you for merging, I got distracted with a test and didn't come back to merge after travis.

**Comment by sthirugn** [comment]
np, ty for checking.

**Comment by elyezer** [comment]
Karma given

**Pull request merged**